Lib/test/test_genericalias.py

```diff
@@ -154,13 +154,11 @@ class MyList(list):
         self.assertEqual(t.__args__, (int,))
         self.assertEqual(t.__parameters__, ())
 
-    # TODO: RUSTPYTHON
-    @unittest.expectedFailure
     def test_repr(self):
         class MyList(list):
             pass
         self.assertEqual(repr(list[str]), 'list[str]')
-        self.assertEqual(repr(list[()]), 'list[()]')
+        self.assertEqual(repr(list[()]), 'list[()]')  

```

Comment:
whitespace is added.

```suggestion
        self.assertEqual(repr(list[()]), 'list[()]')

```

Reply:
@youknowone i fixed it